ui/pages/add-collectible/add-collectible.component.js

```diff
@@ -1,6 +1,7 @@
 import React, { useState } from 'react';
 import { useHistory } from 'react-router-dom';
 import { useDispatch } from 'react-redux';
+import { isValidHexAddress } from '@metamask/controllers/dist/util';
```

Comment:
I'm wondering if it's a good idea to encourage diving into the file tree for `controllers`. It seems like we ought to encourage using the default exports for the module instead. Besides this, above you say `import { util } from '@metamask/controllers`. Does it make sense to reuse that style?